class.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './my_enumerable'
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] as the previous CR mentioned implement a method #each in Mylist that yields successive members of @list and uses the MyEnumerable module. this means that you should create your own #each method using a loop.

Reply:
Thank you for your help, I have used a while loop to create the #each method.